Lib/dbm/sqlite3.py

```diff
@@ -0,0 +1,117 @@
+import os
+import sqlite3
+from contextlib import suppress, closing
+from collections.abc import MutableMapping
+
+BUILD_TABLE = """
+  CREATE TABLE IF NOT EXISTS Dict (
+    key TEXT NOT NULL,
+    value BLOB NOT NULL,
+    PRIMARY KEY (key)
+  )
+"""
+GET_SIZE = "SELECT COUNT (key) FROM Dict"
+LOOKUP_KEY = "SELECT value FROM Dict WHERE key = ? LIMIT 1"
```

Comment:
Hmm, since the key is the primary key, IMO, it should guarantee to return just one.

Reply:
True that

Reply:
```suggestion
LOOKUP_KEY = "SELECT value FROM Dict WHERE key = ?"
```